pr_agent/git_providers/gitlab_provider.py

```diff
@@ -122,6 +134,22 @@ def get_diff_files(self) -> list[FilePatchInfo]:
                                   old_filename=None if diff['old_path'] == diff['new_path'] else diff['old_path']))
         self.diff_files = diff_files
         return diff_files
+    
+    def add_eyes_reaction(self, issue_comment_id: int) -> Optional[int]:
+        try:
+            reaction = self.pr.get_issue_comment(issue_comment_id).create_reaction("eyes")
+            return reaction.id
+        except Exception as e:
+            logging.exception(f"Failed to add eyes reaction, error: {e}")
+            return None
+
+    def remove_reaction(self, issue_comment_id: int, reaction_id: int) -> bool:
```

Comment:
**Suggestion:** Proposed documentation
```suggestion
    def remove_reaction(self, issue_comment_id: int, reaction_id: int) -> bool:
    """
    This method removes a specific reaction from a specific issue comment. It returns True if successful, 
    otherwise it logs the exception and returns False.
    """
```